applications/FemToDemApplication/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ set( KRATOS_FEM_TO_DEM_APPLICATION_SOURCES
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/femdem3d_hexahedron_element.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/extend_pressure_condition_process.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/assign_pressure_id_process.cpp
+	${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/expand_wet_nodes_process.cpp

```

Comment:
Wet...

Reply:
was Francisco Zárate idea haha (it means that is in contact with the pressurfe load)